vb-fe/src/util/Room.ts

```diff
@@ -6,10 +6,30 @@ export interface RoomForm {
   tags: Tag[];
 }
 
+export interface dataCreateRoom {
```

Comment:
Capitalize the first character of the interface

Reply:
can change, one thing we did at gumroad was for all API data, we preluded the interface/type with `data` in lowercase, followed by whatever we wanted to actually name it.  e.g., `dataCreateRoom`

Reply:
ahh i see, but like it idn't a good practice to do that i feel. class names and interfaces always should be capitalized

Reply:
I think preluding it with lowercase is probably just to distinguish it from other types, if I had to guess.

any sort of approach you want to follow for naming things that are coming from apis? preluding it with `Data` or something?  Otherwise will just call it `ICreateRoom`

Reply:
I think just having `Room` would be fine. There should be no difference in the the interface that is coming from the APIs vs the interfaces that resemble the object on the FE. And also it makes it re-usable. For example if you called it DataCreateRoom, this would mean that this type of object is only for an object that is returned by the server from a create room route. Even though the room object, or the list of rooms will have the same interface. So I think it'll be the best if we keep interface names generic